File: TicTacToe_Java_Python/sprint3_0/test/test_board_gui.py
import unittest

from sprint3_0.product.gui import GUI


class TestBoardGUI(unittest.TestCase):
    global board

    def setUp(self):
        pass

    def tearDown(self):
        pass

    def testEmptyBoard(self):
        global board
        board = GUI()
        board.drawEmptyBoard()

    def testNonEmptyBoard(self):
        global board
        board = GUI()
        board.drawNonEmptyBoard()


if __name__ == "__main__":
    unittest.main()

File: TicTacToe/src/sprint2_0/test/TestBoardGUI.java
package sprint2_0.test;

import org.junit.After;
import org.junit.Before;
import org.junit.Test;

import sprint2_0.product.Board;
import sprint2_0.product.GUI;

public class TestBoardGUI {
	private Board board;
	@Before
	public void setUp() throws Exception {
		board = new Board();
	}

	@After
	public void tearDown() throws Exception {
	}

	@Test
	public void testEmptyBoard() {	
		new GUI(board); 
		try {
			Thread.sleep(2000);
		} catch (InterruptedException e) {
			e.printStackTrace();
		}
	}
	
	@Test
	public void testNonEmptyBoard() {
		board.makeMove(0, 0);
		board.makeMove(1, 1);		
		new GUI(board); 
		try {
			Thread.sleep(2000);
		} catch (InterruptedException e) {
			e.printStackTrace();
		}
	}

}
